source/material_model/steinberger.cc

```diff
@@ -220,6 +220,46 @@ namespace aspect
 
 
 
+    template <int dim>
+    double
+    Steinberger<dim>::
+    thermal_conductivity (const double temperature,
+                          const double pressure,
+                          const Point<dim> &position) const
+    {
+      if (conductivity_formulation == constant)
+        return thermal_conductivity_value;
+
+      else if (conductivity_formulation == p_T_dependent)
+        {
+          // Find the conductivity layer that corresponds to the depth of the evaluation point.
+          const double depth = this->get_geometry_model().depth(position);
+          unsigned int layer_index = std::distance(conductivity_transition_depths.begin(),
+                                                   std::lower_bound(conductivity_transition_depths.begin(),conductivity_transition_depths.end(), depth));
+
+          // Make sure the temperature is positive.
+          const double T = std::max(temperature, 1.e-3 * conductivity_reference_temperatures[layer_index]);
```

Comment:
Shouldnt this then be:
```suggestion
          const double T = std::max(temperature, 0.0);
```
Why did you choose 1e-3 * ref_temp?

Reply:
We can choose a different number, but the temperature needs to be positive, it can't be zero, because we divide by this temperature in the lines below. So if I choose a really small number, then the conductivity will get some crazy high value, and I thought that might become a problem for the model (for example, in our 3D models the temperature is sometimes below zero due to over- and undershoots, and we don't want the conductivity to be 1e200 is these cases). So I picked something that would give us a conductivity around 3 orders of magnitude larger than at the reference temperature (which is probably already more than what anyone would want in terms of variations). It's not great, but I don;t have a better idea. Instead, we could also put a maximum conductivity in the input file (and then set it to that value if T is small). 

But let me know if you have a better idea!